src/shared/sentry/sentryadapter.cpp

```diff
@@ -168,3 +157,34 @@ void SentryAdapter::declineCrashReporting() {
   m_userConsent = UserConsentResult::Forbidden;
   emit userConsentChanged();
 }
+
+void SentryAdapter::captureQMLStacktrace(const char* description) {
+  auto engine = QmlEngineHolder::instance()->engine();
+  auto privateEngine = QQmlEnginePrivate::get(engine);
+  QV4::ExecutionEngine* qv4Engine = privateEngine->v4engine();
+  QVector<QV4::StackFrame> stackTrace = qv4Engine->stackTrace(15);
```

Comment:
Very cool! For my information, is this mechanism to get the QML stack frames documented somewhere?